### DISCUSSION TITLE
Confused About Licence

### DISCUSSION THREAD

**Discussion by tbpaksoy** [CGAL/cgal#8617] [Q&A] [answered]
Hello, I would like to make a free project with public repo in GitHub. Do I need to pay anything ? Do my projects needs commercial licence ? Thanks in advance!

**Comment by sloriot** [comment, marked as answer]
If you can respect the term of the GPL v3+ license no commercial license is needed. So basically, if your own project is open source, with a license compatible with GPL v3+, then it's all good.
